AppMovieDB/AppCoordinator.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+import UIKit
+
+class AppCoordinator {
+    var window: UIWindow?
```

Comment:
Should add break line in above and behind this command to separate between property and method of class